### PR TITLE
10217 test

### PR DESCRIPTION
**Pull request by ujahio**

### CODE REVIEW

File: shared/src/business/useCases/filePetitionFromPaperInteractor.ts [hunk 1]
```diff
@@ -5,18 +5,12 @@ import {
...
-  applicationForWaiverOfFilingFeeFile?: Blob;
   applicationForWaiverOfFilingFeeUploadProgress?: (progressEvent: any) => void;
   atpUploadProgress?: (progressEvent: any) => void;
-  corporateDisclosureFile?: Blob;
   corporateDisclosureUploadProgress?: (progressEvent: any) => void;
-  petitionFile: Blob;
   petitionMetadata: PaperCaseDataType;
-  petitionUploadProgress?: (progressEvent: any) => void;
-  requestForPlaceOfTrialFile?: Blob;
+  petitionUploadProgress: (progressEvent: any) => void; // change type
```
Comment (jasonfu9966) [inline shared/src/business/useCases/filePetitionFromPaperInteractor.ts]: we're probably good to keep this

File: web-client/src/presenter/actions/setProgressForFileUploadAction.ts [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+  const startTime = formatNow();
+  const sizeOfFiles: Record<string, number> = {};
+
+  Object.keys(files).forEach(key => {
+    if (!files[key]) return;
+    if (Array.isArray(files[key])) {
+      sizeOfFiles[key] = files[key][0].size;
+      //     files[key].forEach((file, index) => {
+      //       totalSizes[`${key}-${index}`] = file.size;
+      //     });
```
Comment (jasonfu9966) [inline web-client/src/presenter/actions/setProgressForFileUploadAction.ts]: for discussion again: should these comments be removed?

File: web-client/src/views/PetitionQcScanBatchPreviewer.tsx [hunk 3]
```diff
@@ -462,8 +456,8 @@ export const PetitionQcScanBatchPreviewer = connect(
                       />
                     )
                   }
-                  key={`${documentTab.eventCode}_${documentTab.documentId}`}
-                  tabName={`${documentTab.tabTitle}_${documentTab.documentId}`}
+                  key={documentTab.documentType}
+                  tabName={documentTab.fileName} // may need some refactor (established in constants)
```
Comment (jasonfu9966) [inline web-client/src/views/PetitionQcScanBatchPreviewer.tsx]: let's take a look at this some time

File: shared/src/business/entities/EntityConstants.ts [hunk 4]
```diff
@@ -638,35 +638,46 @@ export const INITIAL_DOCUMENT_TYPES = {
...
+    fileName: 'requestForPlaceOfTrialFile',
+  },
+  stin: {
+    documentType: 'Statement of Taxpayer Identification',
+    eventCode: 'STIN',
+    sort: 1,
+    tabTitle: 'STIN',
+    fileName: 'stinFile',
   },
-  stin: STIN_DOCKET_ENTRY_TYPE,
```
Comment (nechama-krigsman) [inline shared/src/business/entities/EntityConstants.ts:663]: Can STIN_DOCKET_ENTRY_TYPE be removed?

File: cypress/cypress-integration/integration/practitioner-request-access-to-case.cy.ts [hunk 5]
```diff
@@ -7,8 +7,8 @@ import {
   loginAsPrivatePractitioner,
 } from '../../helpers/auth/login-as-helpers';
 import {
-  petitionerCreatesEletronicCase,
-  petitionerCreatesEletronicCaseWithDeseasedSpouse,
+  petitionerCreatesElectronicCase,
+  petitionerCreatesElectronicCaseWithDeceasedSpouse,
```
Comment (TomElliottFlexion) [inline cypress/cypress-integration/integration/practitioner-request-access-to-case.cy.ts:11]: Thank. You.

File: web-client/src/presenter/actions/getDocumentSelectedForPreviewAction.ts [hunk 6]
```diff
@@ -3,20 +3,7 @@ import { state } from '@web-client/presenter/app.cerebral';
 export const getDocumentSelectedForPreviewAction = ({
   applicationContext,
   get,
-  props,
-}: ActionProps<{
-  documentId?: string;
-}>) => {
-  const { documentId } = props;
-
-  if (documentId) {
```
Comment (nechama-krigsman) [inline web-client/src/presenter/actions/getDocumentSelectedForPreviewAction.ts:12]: Why was this removed?

File: web-client/src/presenter/actions/setProgressForFileUploadAction.ts (hunk 2 above)
Comment (nechama-krigsman) [reply web-client/src/presenter/actions/setProgressForFileUploadAction.ts]: Yeah, i think we can remove the array logic. It was removed everywhere else

**Review by nechama-krigsman — APPROVED** [review]

File: web-client/src/presenter/actions/getDocumentSelectedForPreviewAction.ts (hunk 6 above)
Comment (ujahio) [reply web-client/src/presenter/actions/getDocumentSelectedForPreviewAction.ts:12]: We no longer need to use the documentId to fetch the document, so this refactor reverts to the original implementation/design of using the tab's file name to fetch the document (docket entry).
